doc/docs.md

```diff
@@ -6920,6 +6921,20 @@ with identical hardware.
 
 - When distributing the software to users with potentially older CPUs.
 
+#### `-compress`
+
+This flag executes `upx` to compress the resultant executable,
+reducing its size by around 50%-70%.
+
+**When to Use**
+
+- For really tiny envs that the size it's a first, but at runtime everything will run as
+if it had no compression.
```

Comment:
```suggestion
- For really tiny environments, where the size of the executable on the file system, or when deploying is important. The executable will be uncompressed at runtime, so it will take a bit more time to start.
```